rustls/src/lib.rs

```diff
@@ -496,9 +495,9 @@ pub mod version {
 ///
 /// ALL_KX_GROUPS is provided as an array of all of these values.
 pub mod kx_group {
-    pub use crate::kx::SECP256R1;
-    pub use crate::kx::SECP384R1;
-    pub use crate::kx::X25519;
+    pub use crate::crypto::ring::SECP256R1;
```

Comment:
Arguably this `kx_group` module should itself move into `crypto::ring`? (Probably as a separate commit though.)

Reply:
I moved this in 495469b4c640e781f433441ba04d65bd0680b4a3 but left a `pub use` in the root `lib.rs`. If you were thinking we should remove all traces of this from the top-level API I can touch that up.